gen.py

```diff
@@ -112,6 +113,25 @@ class EmptyTopLevelPlotFlags(BaseModel):
     __root__: None = None
 
 
+class TopLevelArtifactFlags(BaseModel):
+    type: str = Field(None, description="Type of the artifact")
+    path: str = Field(None, description="Path to the artifact")
+    desc: str = Field(None, description="Description for the artifact")
+    meta: dict[str, Any] = Field(
+        None, description="Custom metadata for the artifact"
+    )
+    labels: set[str] = Field(
+        default_factory=set, description="Labels for the artifact"
+    )
+
+    class Config:
+        extra = "forbid"
+
+
+class EmptyTopLevelArtifactFlags(BaseModel):
+    __root__: None = None
```

Comment:
Not sure I follow why we need this @aguschin , could you explain?

Reply:
This is needed to support the following dvc.yaml:
```
artifacts:
    myart:
```

Without it, it would need to look like
```
artifacts:
    myart: {}
```

Reply:
resolved - since `path` is required, this is no longer relevant. Removed.